akka-actor-tests/src/test/scala/akka/AkkaExceptionSpec.scala

```diff
@@ -16,6 +16,9 @@ import akka.actor._
  */
 class AkkaExceptionSpec extends AnyWordSpec with Matchers {
 
+  // TODO DOTTY, yes I know it's wrong
+  implicit val pos: org.scalactic.source.Position = new org.scalactic.source.Position(fileName = "", filePathname = "", lineNumber = 1)
```

Comment:
This is a workaround, ScalaTest is usable with Scala 3, but I found a few rough edges.
The implicit position is provided from a macro in Scala 2.X and is not available in Scala 3.

I see that there is some ongoing work to cross-compile ScalaTest (e.g. https://github.com/scalatest/scalatest/commit/e8201d42e5c1ba61c82c0770906a6fdd2e593cd8 ) and I expect that this issue is solved there, maybe @bvenners or @cheeseng can chip in?

Reference issue:
https://github.com/scalatest/scalatest/issues/1950